From 1b749cc64a7cb3eedcdd3fada149dd941d1beecd Mon Sep 17 00:00:00 2001
From: Ley Foon Tan <ley.foon.tan@intel.com>
Date: Mon, 23 Mar 2020 18:40:32 +0800
Subject: [PATCH 115/740] HSD #1507875164-2: fpga: socfpga: Add call to
 socfpga_sdram_apply_static_cfg()

Add call to socfpga_sdram_apply_static_cfg() function after write to SDR
register.

Signed-off-by: Ley Foon Tan <ley.foon.tan@intel.com>
---
 drivers/fpga/socfpga_gen5.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/fpga/socfpga_gen5.c b/drivers/fpga/socfpga_gen5.c
index d73474f29e..4dbe3ec924 100644
--- a/drivers/fpga/socfpga_gen5.c
+++ b/drivers/fpga/socfpga_gen5.c
@@ -8,6 +8,7 @@
 #include <asm/io.h>
 #include <linux/errno.h>
 #include <asm/arch/fpga_manager.h>
+#include <asm/arch/misc.h>
 #include <asm/arch/reset_manager.h>
 #include <asm/arch/system_manager.h>
 
@@ -217,6 +218,7 @@ int socfpga_load(Altera_desc *desc, const void *rbf_data, size_t rbf_size)
 	/* Disable all signals from FPGA to HPS SDRAM */
 #define SDR_CTRLGRP_FPGAPORTRST_ADDRESS	0x5080
 	writel(0, SOCFPGA_SDR_ADDRESS + SDR_CTRLGRP_FPGAPORTRST_ADDRESS);
+	socfpga_sdram_apply_static_cfg();
 
 	/* Disable all axi bridge (hps2fpga, lwhps2fpga & fpga2hps) */
 	socfpga_bridges_reset(1);
-- 
2.43.0

